src/App.js

```diff
@@ -23,21 +24,37 @@ const dummyMovie =  {id: 1,
 
 function App() {
 
-  const [movies, setMovies] = useState(movieData.movies)
+  const [movies, setMovies] = useState([])
   const [singleMovie, setSingleMovie] = useState(null)
 
-  function getMovieDetail() {
-    // i think the fetch will live here - fetch('url {id}')
-      setSingleMovie(dummyMovie)
-      // console.log(movies, "movies")
+  function getAllMovieData(){
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+    .then(data => setMovies(data.movies))
+    .catch(error => alert(`${error} please try again later`))
+  }
+
+  function getMovieDetail(id) {
+    fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+    .then(response => response.json())
+    .then(data => setSingleMovie(data.movie))
+    .catch(error => alert(`${error} sorry try again later`))
   }
+
   console.log("movies", movies)
```

Comment:
Nice job using console logs for debugging! To further improve your code, make sure to remove all console logs before finalizing your submission.